CONTRIBUTING.md

```diff
@@ -13,13 +13,37 @@ pnpm run setup
 
 Note for Windows: Use the latest version of [Git Bash](https://gitforwindows.org/)
 
+### General prerequisites
+
+1. Install [`pnpm`](https://pnpm.js.org/) (for installing npm dependencies)
+1. Install [`docker`](https://www.docker.com/products/docker-desktop) (for managing test databases)
```

Comment:
We could add install `ts-node`I guess

Reply:
Maybe missed something but didn't see any raw ts-node invocation use-cases for the project?

Reply:
Line 19 below we have `1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile``